docs/docs/admin/teamforcapella/migration.md

```diff
@@ -36,8 +41,19 @@
        on the old TeamForCapella server.
        ![Disable backup option](../../user/tools/capella/migration/backup-migration.png)
 
-1.  Export the model to the new TeamForCapella server via the Export wizzard
+1.  Export the model to the new TeamForCapella server via the Export wizard
     from TeamForCapella:
```

Comment:
Note that all images must explicitely be selected for export, example:

<img width="604" alt="image" src="https://github.com/user-attachments/assets/24562359-4e0b-4e62-8df2-10c9b02abfc3">

Reply:
Avoid to share DBS internal information like internal project names on GitHub, I've removed the picture.

Reply:
In any case, this is already described on the Export to TeamForCapella repository page: 
![image](https://github.com/user-attachments/assets/3a143d64-43a7-44e1-a392-f3b61bf47997)